Makefile

```diff
@@ -417,3 +417,23 @@ generate-labels:
 		gh label create "$${TYPE}/$${NAME}" -c "$${COLOR}"; \
 	done; \
 	exit 0
+
+.PHONY: generate-all-component-dropdowns
+generate-all-component-dropdowns:
```

Comment:
The name is a bit unclear in outside of the github issues context. Maybe we can have one target called `generate-gh-issue-templates` with a loop `for FILE in bug_report feature_request other; do \`?

Reply:
Makes sense to me. Done.